intermediate-python-debugging/problem-set-using-the-debugger.md

```diff
@@ -94,9 +94,9 @@ Once you identify the cause of the failure, adjust the code to make the test pas
 
 ### !callout-warning
 
-## Discover Tests to Show the Test Status Icons
+## Getting VS Code to Notice Your Tests
 
-If you don't see status icons to left of the test names in your test file, you may need to let VS Code _discover_ your tests!  Expand the section below for a review of how to do this!
+If you don't see status icons to left of the test names in your test file, you may need to verify that your virtual environment is being used or that the testing framework has been configured!  Expand the section below for a review of how to do this!
```

Comment:
Aligned with my suggestion below, what if we updated this callout as follows, and cut the whole detail section?
```suggestion
If you don't see status icons to left of the test names in your test file, you may need to verify that your virtual environment is being used or that the testing framework has been configured! You can review the steps to do so in the previous lesson called "Debugging Tests with Pytest".
```